web-dev-cody-hackathon/100editors
# Updated metadata

## Description

Closes #16

This pull request addresses the following changes to the `layout.tsx` file:
Update metadata object in `layout.tsx` to include [describe what changes you made to the metadata object].

## Testing

To test this code, please follow these steps:

1. [x] Checkout this branch locally: `git checkout branch-name-here`
2. [x] Run the application and navigate to pages that use the `layout.tsx` file.
3. [x] Verify that the updated metadata object displays the correct information and that it functions as expected.

## Type of change

- [ x ] Other (please specify): Metadata change

## Screenshots

N/A

## Checklist

- [x] Changes generate no new errors or warnings
